src/genome_builds.php

```diff
@@ -244,7 +244,193 @@
+        if ($bValidPassword) {
+            $sColumnSuffix = $sID;
+
+            // Remove genome build from database.
+            $_DB->query('DELETE FROM ' . TABLE_GENOME_BUILDS .
+                        ' WHERE id = "' . $sID . '"');
+
+            // Prepare an array to more easily remove the columns from the
+            //  VOG and transcripts tables.
+            $aTablesAndTheirColumns = array(
+                TABLE_VARIANTS => array(
+                    'VariantOnGenome/DNA/' . $sColumnSuffix,
+                    'position_g_start_' . $sColumnSuffix,
+                    'position_g_end_' . $sColumnSuffix,
+                ),
+                TABLE_TRANSCRIPTS => array(
+                    'position_g_mrna_start_' . $sColumnSuffix,
+                    'position_g_mrna_end_' . $sColumnSuffix,
+                ),
+            );
+
+            // Remove the columns to the VOG and Transcripts tables.
+            foreach ($aTablesAndTheirColumns as $sTable => $aTable) {
+                $aActiveColumns = $_DB->query('
+                    SELECT COLUMN_NAME
+                    FROM information_schema.COLUMNS
+                    WHERE TABLE_SCHEMA = DATABASE()
+                      AND TABLE_NAME = ?
+                      AND COLUMN_NAME IN (' . implode(',', array_fill(0, count($aTable), '?')) . ')',
+                    array_merge(array($sTable), $aTable))->fetchAllColumn();
+
+                $bToRemove = false;
+                $sSQL = 'ALTER TABLE ' . $sTable;
+
+                foreach ($aTable as $sColumn) {
+                    if (in_array($sColumn, $aActiveColumns)) {
+                        $bToRemove = true;
+                        $sSQL .= ' DROP COLUMN `' . $sColumn . '`,';
+                    }
+                }
+
+                if ($bToRemove) {
+                    $_DB->query(rtrim($sSQL, ','));
+                }
+            }
+
+            // Deactivate custom DNA column.
+            $_DB->query('DELETE FROM ' . TABLE_ACTIVE_COLS .
+                ' WHERE colid = "VariantOnGenome/DNA/' . $sID . '"');
+
+
+            // Write to log...
+            lovd_writeLog('Event', LOG_EVENT, 'Removed Genome Build ' . $sID);
+
+            // Thank the user, and send them to the page of the currently active GBs.
+            header('Refresh: 5; url=' . lovd_getInstallURL() . 'genome_builds');
+
+            $_T->printHeader();
+            $_T->printTitle();
+            lovd_showInfoTable('Successfully removed the chosen genome build!', 'success');
+            $_T->printFooter();
+            exit;
+        }
+    }
+
+
+
+    $_T->printHeader();
+    $_T->printTitle();
+
+    $sName = $sID . ' / ' . $_SETT['human_builds'][$sID]['ncbi_name'];
+
+    lovd_showInfoTable('This will deactivate genome build ' . $sName . ' from your database.', 'warning');
+
+    lovd_errorPrint();
+
+    // Tooltip JS code.
+    lovd_includeJS('inc-js-tooltip.php');
```

Comment:
This is not wrong, but simply unneeded since this form doesn't have any tooltips. So I chose to remove it to prevent a file from getting included that wasn't used at all.